packages/auth/src/app.ts

```diff
@@ -206,7 +206,7 @@ export class App {
     const openApi = await this.container.use('openApi')
     /* Back-channel GNAP Routes */
     // Grant Initiation
-    this.publicRouter.post(
+    this.publicRouter.post<DefaultState, CreateContext>(
```

Comment:
We need these for when there is a `body` defined on `Context`s request, since that is the case where the `body: unknown` clashes with the given `Context`'s. e.g. `ManagementContext` just has `params`.

Going to do the `backend` types in separate PR

Reply:
I was trying to understand why `backend` seems fine.
It appears to be due to:
- the generic route definition in the magic `for` loop
https://github.com/interledger/rafiki/blob/e773fb7cd533a1a8495e056987a772154651130b/packages/backend/src/app.ts#L331-L341
- the use of `ts-ignore`
https://github.com/interledger/rafiki/blob/e773fb7cd533a1a8495e056987a772154651130b/packages/backend/src/app.ts#L369-L370

I got errors similar to what `<DefaultState, CreateContext>` here avoids by explicitly defining a route outside of the magic `for` loop as well as removing a `ts-ignore`

I've opened:
- https://github.com/interledger/rafiki/issues/916
- https://github.com/interledger/rafiki/issues/917